bundler-parceler/src/main/java/com/hannesdorfmann/fragmentargs/bundler/ParcelerArgsBundler.java

```diff
@@ -15,7 +15,7 @@
     bundle.putParcelable(key, Parcels.wrap(value));
   }
 
-  @Override public <V> V get(String key, Bundle bundle) {
+  @Override public Object get(String key, Bundle bundle) {
```

Comment:
Are you sure that this is working properly? I mean, if we return here `Object` somewhere has to be a cast to the real class, right? By using generic methods `<V>` the compiler can determine what the actual return type is because `Parcels.unwrap()` knows `<V>` too.